topi/python/topi/bifrost/conv2d.py

```diff
@@ -370,12 +371,7 @@ def _schedule_winograd(cfg, s, op):
         s[G].compute_inline()
         eps, _, _, _ = s[U].op.axis
         y, _, _, _ = s[padded_kernel].op.axis
-        if autotvm.GLOBAL_SCOPE.in_tuning:
-            # Kernel transformation will be pre-computed during compilation, so we skip
-            # this part to make tuning records correct
-            s[U].pragma(eps, 'debug_skip_region')
-            s[padded_kernel].pragma(y, 'debug_skip_region')
-        else:
+        if not autotvm.GLOBAL_SCOPE.in_tuning:
```

Comment:
Also winograd kernel transformation is different from spatial_pack. We need to have a different path to generate placeholder.

Reply:
Yes. I put the placeholder in `_decl_winograd` to biforst/con2d.py and mali/con2d.py